rails-questions.md

```diff
@@ -1,30 +1,52 @@
 # ASSESSMENT 5: Database and Active Record Practical Questions
 
-1. What is SQL?
+### 1. What is SQL?
 
+SQL (Structured Query Language) is a domain-specific language designed to manage data in a relational database. PostgreSQL, MySQL, and SQLite are different database management systems that share the same basic language.
 
-2. What the PostgreSQL query that would return all the content in a particular table?
+### 2. What the PostgreSQL query that would return all the content in a particular table?
 
+```sql
+SELECT *
+FROM table
+```
 
-3. What is the command to create a new Rails application with a PostgreSQL database?
+### 3. What is the command to create a new Rails application with a PostgreSQL database?
 
+```ruby
+rails new application_name -d postgresql
+```
 
-4. What is the command to generate a Rails model for a meals table with columns named breakfast, lunch, and dinner?
+### 4. What is the command to generate a Rails model for a meals table with columns named breakfast, lunch, and dinner?
 
+```ruby
+rails g model Meal breakfast lunch dinner
+```
 
-5. What is a migration? Why would you use one?
+### 5. What is a migration? Why would you use one?
 
+a migration is a sort of instruction that you can use to instruct rails in how to create and update a database.
```

Comment:
Yea, technically a migration is a file that tracks the history of database schema changes.